repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorHandler.java

```diff
@@ -153,9 +146,16 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 }
 
                 if (!isValid) {
-                    if (mbcsInvalidRequests != null) {
+                    if (Optional.ofNullable(metricsService).isPresent()) {
```

Comment:
Cool. That sounds like a winner. Removing all of the `if (Optional.ofNullable(metricsService).isPresent()) {` in the Java files along with the `val metricsServiceOption = Option(metricsService)` and `metricsServiceOption.foreach(metricsService =>` in the Scala files should fix it.